File: app/src/main/java/com/pnuema/android/bible/ui/viewholders/VerseViewHolder.java
package com.pnuema.android.bible.ui.viewholders;

import android.view.LayoutInflater;
import android.view.ViewGroup;
import android.widget.TextView;

import com.pnuema.android.bible.R;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class VerseViewHolder extends RecyclerView.ViewHolder {
    private TextView verseText;

    public VerseViewHolder(@NonNull final ViewGroup parent) {
        super(LayoutInflater.from(parent.getContext()).inflate(getLayout(), parent, false));

        verseText = itemView.findViewById(R.id.verseText);
    }

    public void bind(final CharSequence text) {
        verseText.setText(text);
    }

    public static int getType() {
        return getLayout();
    }

    private static int getLayout() {
        return R.layout.listitem_verse;
    }
}

File: app/src/main/java/com/pnuema/android/bible/ui/viewholders/VerseViewHolder.kt
package com.pnuema.android.bible.ui.viewholders

import android.view.LayoutInflater
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.pnuema.android.bible.R

class VerseViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(LayoutInflater.from(parent.context).inflate(layout, parent, false)) {
    private val verseText: TextView = itemView.findViewById(R.id.verseText)

    companion object {
        val type: Int
            get() = layout

        private val layout: Int
            get() = R.layout.listitem_verse
    }

    fun bind(text: CharSequence) {
        verseText.text = text
    }
}
